Pino-iOS/Pino-iOS/ViewController.swift

```diff
@@ -9,11 +9,12 @@ import UIKit
 
 class ViewController: UIViewController {
```

Comment:
this class and its storyboard should be deleted, we don't need storyboard in this project. set Intro view as initial view controller.

Reply:
I see I'll take a look at it